src/os_vms_conf.h

```diff
@@ -148,27 +155,51 @@
 #undef  HAVE_ISNAN
 #undef  HAVE_XOS_R_H
 #define HAVE_NO_LONG_LONG
+#define VIM_SIZEOF_INT  4
 #define VIM_SIZEOF_LONG 4
 #define LONG_LONG_MIN  (-2147483647-1)
 #define LONG_LONG_MAX  (2147483647)
 #define ULONG_LONG_MAX (4294967295U)
-#else // AXP and IA64
+
+#else // ALPHA, IA64, X86_64
+#if defined(__alpha) || defined(__ALPHA)
+#if !defined(ALPHA)
+#define ALPHA
```

Comment:
Aren't the definition name `ALPHA`, `IA64` and `X86_64` too generic?
Are they defined by the OpenVMS compiler?

Reply:
because of the explicit restriction by __CRTL_VER_OVERRIDE=80400000 the X86_64 is lost and needed to define for current and future use.
But you are right... defining others like ALPHA is an overkill.
I'll remove them.